bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -63,32 +63,6 @@ spec:
       serviceAccountName: {{ default (include "common.names.fullname" . ) .Values.serviceAccount.name }}
       {{- end }}
       initContainers:
-      {{- if .Values.pgpool.tls.enabled }}
```

Comment:
The chart is configured to use Kubernetes Security Context to automatically change the ownership of the volumes. However, this feature does not fully work in all Kubernetes distributions or you can face some restrictions when you are using it. Also, our images run as non-root by default, so is very important to have the right permissions.

This piece of code allow users tune the permissions in those scenarios